core/src/main/resources/hudson/tools/Messages_de.properties

```diff
@@ -21,20 +21,14 @@
 # THE SOFTWARE.
 
 ToolLocationNodeProperty.displayName=Verzeichnisse von Hilfsprogrammen anpassen
-CommandInstaller.DescriptorImpl.displayName=Shell-Kommando ausf\u00FChren
+CommandInstaller.DescriptorImpl.displayName=Shell-Kommando ausf\u00fchren
```

Comment:
Looks like Netbeans changes the hex characters to lower case: https://netbeans.org/bugzilla/show_bug.cgi?id=239513. I'll clean it up.